api/test/wfl/integration/modules/covid_test.clj

```diff
@@ -1,6 +1,7 @@
+"Test the Sarscov2IlluminaFull COVID pipeline."
 (ns wfl.integration.modules.covid-test
-  "Test the Sarscov2IlluminaFull COVID pipeline."
   (:require [clojure.test :refer :all]
+            [clojure.test :as clj-test]
```

Comment:
This is a subset of `:refer :all` so remove it.

Reply:
Removed